docs/3.0.0-beta.x/advanced/traceability.md

```diff
@@ -1,44 +1,53 @@
-- How long it takes to set up a project? If the time increases, does it means that they are encountering issues or the process is simply too complicated?
+- Are our users using a particular feature or not? For those who are using it, what do they use it for? Is it activated and used along side another plugin? Which specific plugin? Or something else, like, only in development/production?
+- How long does setting up a project take? If the global install time increases, does it means that users are encountering issues or the process is simply too complicated?
 - What type of errors our users are facing?
 - What are the most used plugins?
-- Should we focus our effort on being compatible with Node 12? Even if it is not the most used versions based on Node.js metrics, is it the same for our community?
+- Should we focus our efforts to being compatible with Node 12? Maybe our community uses version 12, in greater percentages than the global Node.js community?
+- And more...
 
-And many more... without these metrics, we won't be able to do the right choice to give you the best experience as possible.
+Without these metrics, we wouldn't be able to make the right choices as we continue to move forward with the roadmap and provide what you, the community and users, are asking for.
 
-
-## What are the collected data?
+### What data is collected?
 
 - Unique project ID (generated with UUID)
 - Unique machine ID (generated with [node-machine-id](https://www.npmjs.com/package/node-machine-id))
 - Environment state (development, staging, production)
-- OS informations (system)
+- System information (OS)
 - Build configurations
 
 ::: warning GDPR
-The collected data are non-sensitive or personal data. We are compliant with the European recommendations (see our [Privacy Policy](https://strapi.io/privacy)). We do not collect databases configurations, password or custom variables. The data are secured, encrypted and anonymized.
+The collected data are of a non-sensitive nature and no personal data is collected. We are compliant with the European GDPR recommendations (see our [Privacy Policy](https://strapi.io/privacy)). We do not collect databases configurations, password or custom variables. Any data collected (as above) is secured, encrypted and anonymized.
 :::
 
+### How to opt-out?
+
+You can easily disable the default data collection feature.
+
+Should you decide to opt-out, you may do so by removing the `uuid` property in the `package.json` file located within the root of your project. This will automatically disable this feature.
+
+```json
+{
+  "name": "my-project",
+  "version": "0.1.0",
+  "strapi": {
+    "uuid": "7b581c0d-89b7-479e-b379-a76ab90b8754"
+  },
+  "license": "MIT"
+}
+```
+
+### Questions?
+
+Should you have any questions regarding this feature, feel free to email [hi@strapi.io](mailto:hi@strapi.io), and we will answer you right away.
```

Comment:
`and we will answer you as soon as possible`

Not a good idea to make "promises" that might be difficult to keep.

Reply:
Agreed. @derrickmehaffy